### PR TITLE
chore: Python notebook updates & additions

### PR DESCRIPTION
**Pull request by danieljhegeman**
- complete R notebooks for all endpoints
- minor updates to headers and descriptions in python notebooks (and some old R notebooks)

### CODE REVIEW

File: notebooks/curation_api/R/delete_collection_R.ipynb [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+   "source": [
+    "print(res$status_code)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Formulate request and create new private Collection"
```
Comment (Bento007) [inline notebooks/curation_api/R/delete_collection_R.ipynb:155]: Should be delete

File: notebooks/curation_api/R/delete_dataset_R.ipynb [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+   "source": [
+    "print(res$status_code)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Formulate request and create new private Collection"
```
Comment (Bento007) [inline notebooks/curation_api/R/delete_dataset_R.ipynb:175]: Should be delete

File: notebooks/curation_api/R/get_collection_R.ipynb [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    "site_url <- str_interp(\"https://${domain_name}\")\n",
+    "api_url_base <- str_interp(\"https://api.${domain_name}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Formulate request and create new private Collection"
```
Comment (Bento007) [inline notebooks/curation_api/R/get_collection_R.ipynb:96]: Should be Get a collection

File: notebooks/curation_api/R/get_collections_R.ipynb [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "74560383",
+   "metadata": {},
+   "source": [
+    "The script in this notebook fetches... \n",
+    "- all public Collections (with query param `visibility=PUBLIC` or not set).\n",
+    "\n",
+    "or\n",
+    "- all access-allowed private Collections (with query param `visibility=PRIVATE`; access token required).\n",
```
Comment (Bento007) [inline notebooks/curation_api/R/get_collections_R.ipynb:20]: all accessible?

File: notebooks/curation_api/R/get_dataset_assets_R.ipynb [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+    "site_url <- str_interp(\"https://${domain_name}\")\n",
+    "api_url_base <- str_interp(\"https://api.${domain_name}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Formulate request and create new private Collection"
```
Comment (Bento007) [inline notebooks/curation_api/R/get_dataset_assets_R.ipynb:116]: get dataset assets

File: notebooks/curation_api/R/get_dataset_R.ipynb [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+    "site_url <- str_interp(\"https://${domain_name}\")\n",
+    "api_url_base <- str_interp(\"https://api.${domain_name}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Formulate request and create new private Collection"
```
Comment (Bento007) [inline notebooks/curation_api/R/get_dataset_R.ipynb:116]: get dataset

File: notebooks/curation_api/R/update_curator_tag_R.ipynb [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+   "source": [
+    "print(res$status_code)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6085d322",
+   "metadata": {},
+   "source": [
+    "### Formulate request and create new private Collection"
```
Comment (Bento007) [inline notebooks/curation_api/R/update_curator_tag_R.ipynb:196]: update curator tag

**Review by Bento007 — CHANGES_REQUESTED** [review]
A few typo to fix

**Comment by Bento007** [comment]
@danieljhegeman merged main back in your branch. Reverse the commit if you need to.

**Review by Bento007 — APPROVED** [review]

File: notebooks/curation_api/R/get_dataset_assets_R.ipynb [hunk 8]
```diff
@@ -0,0 +1,158 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d7371ef",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Enter the identifier (curator tag or Dataset id) of the Dataset whose curator tag you want to update</font>\n",
+    "\n",
+    "_The `dataset_id` and `curator_tag` can both be found using the_ `/collections/{collection_id}` _endpoint and filtering for these attributes on the Dataset of interest._"
```
Comment (ebezzi) [inline notebooks/curation_api/R/get_dataset_assets_R.ipynb:57]: Those `_` don't seem to be working if you look at the render.

File: notebooks/curation_api/R/update_curator_tag_R.ipynb [hunk 9]
```diff
@@ -0,0 +1,241 @@
...
+   "source": [
+    "identifier <- \"the_tag.h5ad\"  # Or use the id: \"01234567-89ab-cdef-0123-456789abcdef\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aeb7861b",
+   "metadata": {},
+   "source": [
+    "<font color='#bc00b0'>(Required) Enter your new chosen curator tag. **Must possess the '.h5ad' suffix**.</font>\n",
```
Comment (ebezzi) [inline notebooks/curation_api/R/update_curator_tag_R.ipynb:93]: I believe this is no longer required?

File: notebooks/curation_api/python/create_dataset_from_local_file.ipynb [hunk 10]
```diff
@@ -13,19 +13,19 @@
...
     "\n",
     "_For **NEW** Dataset uploads_:\n",
     "- You must decide upon a string identifier (a \"curator tag\") to use to uniquely identify the resultant Dataset within its Collection. This tag **MUST NOT** be the tag of an existing Dataset within this Collection (otherwise the existing Dataset will be updated; read on below), and it **MUST NOT** conform to the uuid format. The tag suffix **MUST** be `.h5ad`.\n",
     "\n",
     "_For **replacing/updating** existing Datasets_:\n",
     "- Uploads to a curator tag for which there already exists a Dataset in the given Collection will result in the existing Dataset being replaced by the new Dataset created from the datafile that you \n",
     "are uploading.\n",
-    "- Alternatively, an existing dataset may be targeted for replacement by using the Dataset's Cellxgene uuid as the identifier when writing to S3. In this scenario, the uuid **MUST** have `.h5ad` appended as a suffix.\n",
+    "- Alternatively, an existing dataset may be targeted for replacement by using the Dataset's Cellxgene id as the identifier when writing to S3. In this scenario, the id **MUST** have `.h5ad` appended as a suffix.\n",
```
Comment (ebezzi) [inline notebooks/curation_api/python/create_dataset_from_local_file.ipynb:28]: I think we should use the new format: CELLxGENE

**Review by ebezzi — APPROVED** [review]

File: notebooks/curation_api/python/create_dataset_from_local_file.ipynb (hunk 10 above)
Comment (Bento007) [reply notebooks/curation_api/python/create_dataset_from_local_file.ipynb:28]: I'll leave that as a separate PR for now.

File: notebooks/curation_api/R/update_curator_tag_R.ipynb (hunk 9 above)
Comment (Bento007) [reply notebooks/curation_api/R/update_curator_tag_R.ipynb:93]: This removed in a separate PR

File: notebooks/curation_api/R/get_dataset_assets_R.ipynb (hunk 8 above)
Comment (Bento007) [reply notebooks/curation_api/R/get_dataset_assets_R.ipynb:57]: It renders in my pycharm environment. I will leave it as is and chalk it up to GitHub's problem.

**Pull request merged**